greenback-kit-core/src/main/java/com/greenback/kit/model/AutoExportRun.java

```diff
@@ -0,0 +1,155 @@
+package com.greenback.kit.model;
+
+import java.time.Instant;
+
+public class AutoExportRun extends GreenbackObject{
+    private String referenceId;
+    private AutoExportRunType type;
+    private ProcessingStatus status;
+    private String userId;
+    private Long typeTagId;
```

Comment:
This is just for internal use -- we wouldn't want this exposed in our public API.